frontend/src/configureStore.spec.ts

```diff
@@ -1,22 +0,0 @@
-import { createStore, applyMiddleware } from 'redux';
```

Comment:
Was getting errors from this that weren't making sense, will circle back later in the sprint.